authzed/api/v1alpha1/lookupwatch_service.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+package authzed.api.v1alpha1;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1alpha1";
+option java_package = "com.authzed.api.v1alpha1";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v1/core.proto";
+
+// LookupWatchService is used to receive a stream of updates for resources of a
+// specific (resource type, permission, subject) combination.
+service LookupWatchService {
```

Comment:
This will eventually live in the v1.WatchService so let's make this parallel that.

Reply:
@josephschorr this is contradictory to what we settled on https://github.com/authzed/spicedb/issues/207.

I remember proposing this in the LookupWatch API proposal. It always made sense to me for this to be part of the Watch API, but I thought we agreed to put it under a difference service specification. Has that decision just changed?

Reply:
I'd prefer it being WatchService rather than LookupWatchService. WDYT @josephschorr? When it moves into v1 it certainly should be.

Reply:
I prefer we keep it distinct, as I can absolutely imagine scenarios where LookupWatchService is exposed to applications using SpiceDB but Watch is not; for example, an application might use LookupWatchService to track changes directly, but the admin of the SpiceDB cluster may only want "managed" users of WatchService to be exposed (future caches, audit logging, etc)

Reply:
I think this makes sense, let's call it "WatchResourcesService" then and call this good to go.

Reply:
Cool. I've updated the service name and renamed the file to match it.